models/nft/ethereum/nft_ethereum_floor_price.sql

```diff
@@ -0,0 +1,126 @@
+{{ config(
+    alias = 'floor_price',
```

Comment:
one good point of feedback from @0xRobin: this is a good base for a collection stats table, so I would rename the spell to `collection_stats`

Reply:
ty ser, I'll add tests later tonight and update the naming.

Reply:
tests added and name updated